test/e2e/mcd_test.go

```diff
@@ -1122,3 +1122,197 @@ func setupForInternalImageRegistryPullSecretTest(t *testing.T, cs *framework.Cli
lZgLzE8pudfXUw/0R7dzEviVpE4nCXxCsw4etIkw8ZSHPegPiOJ2CIJgJFJRdLcv5D28I8Eur6VbMrhiA6R0rTH8tuNTaIyC6WhISs6louUhylpdkF/TIVbGyBzGBOyTBrhPp5C2kGnpJO0ZrF6c/Mgqq0WjIdYhNpvh4zIPPZP17BVFHWlJO+aY2AEwq5lVRPPY0XA0QD0s4u1alKB3XiO6jVdB1ibT3AlSrl50ftRoVeD5Wi2y2jixRldiA4Bii1NaUSufeCmVyHnTl/vCey/YHbAQZBT0ktETySTGcHRkGyIfnoA0mXZZVMAVjCUCAvUeZvRnWQZb8sI1k4t9Ctam68deAJCnSrgYFFsifSCqgJwOugA+C0bWjwr1k85y8mOAKULKj6XMp9uPBGCut8tAT8v34hP1P2hCdpjVrFLX0hrFWb0tYNVke99iGNT77JB8eLcfs9/fhplthESAY3hWssCWPDxII7IAjJmY4R6mtFPwCVkIsOk3O2cg8eZr/BAXtezSVw+2QC4+a3cl3+Rbhkl9SHjLPB/EvsDANxAv5vyIn1eQfBBLuJzvJWwuTDvhIYhBXi77xKLdNRh8P++GCbh5YdHgH4NLeSHrh59gvU9j66nsyhZaLGIbmX5/YOJBZ0Ucaio8DEj18LPG5gAuh3TctKx7eYR9/27oesbPRViOfGY87LEEJKWwd/1eD/A48j96MawPJqyTg2JkLIhHNg7MUeTWRZxMaNUspTBq8/ef17/Q3dHL9AQ22aLFWfQOGcuz4NH/0gvKjkNQIoB554fcEIG3mMuos28Hw8XcaDrvaQZhNIA4co/7nNVBenB7YdGVmno/WMlavA5+cdlmF+Kfkj/5VAwfYNsP6HKqw8FTmoNu6pWv0MYFjO/na18lsHO91EkCVP9ZHx6GeInuN3B2lzhk7b9XN6V1+nap7ZNArAGMwRdM5z5EKOKObY6nM2X2C1hso4/slFB5IWfHgkE5K+VAGNHUIkTCwGD/i8AISLnNu1Uu0NuZWdGxQOQbwvx85bFLtMNCdT/Gf5iEqTTLr6flRqaiwl+d5Ovx3AAGhVIA7JTgKzfcJFf/mgXRoTxH3gnbvye/k4pv9b4O1GDOzv8rhZ31GtKHfxqQAEyTkTpStur0Qul6u16TnEQqiET0oE1dgwgk7/jlmQvILBc9rcAuxDOKW6HTbUkIGgo+Cr9HicHy76stzRX2U3sZAZK3ObsIGDztHCWb9y2ydL5Z8nYdWXaZQEZwB2c8v/wl0PvWQL+eJwzfwqD6xVZ3UnanI6i8wH1ng0oOHAOe8+6+SkqJPFbHQsd8q2hQgTeYlwvxVHm6VX6WSw+nM8Wxy+VJqIAQSwX+wdmUIByBPA1OZWdyAMG21KNnj4bVODQMvlLmRUTm9/E9XJ/kFJpMNNXTbAovo6Hs77aNBLyOJxx2UlDF+2kvIU6gQZMwwoM+nw5GAkqVGHnerKjtmn/S6ThDj94DfsWLXO+oWa/l6mDxEO9EAKQ8rWEAVJ2UWZPkvyWtVhWIZhLqKXFXDz5U/sXAM8kF1C8LUuVmZab9bNhjjNSyD6ClWYwSLJEu0zREcFF1R1X6R+r6T0DYKQQvyr/hokHgasNJGx2Eb+pB8MQdRgLiQ3WBpi0rLH+096T3EPbQ60JHQmxUZU00xXbxqw8NO2YeUx6mbWJf07EDQhH1HYCxmqcftMhynA5BYWT1rB6ldBNb0pkfTHyb0Y1lXn6VBnye053m5QBeoy537mnm0yVDzKCqx0/idimmZiGAf8kU6M9kvH5dpSXWJNNx88P6cPibBZbBpQkf+HMbgiATLNBjL0hyFpgr1tddh5Rz4DQOJe93kWuNpTfMZDKmWJGSN7CTABGDuC+8ir0+iyKqNg8btrLjIXjEdyQ85NCPzNRX/Vzig2Yu57KEZCfA6C8sq9xZVm80VlgWLfn1uZnowIzrTWkYFPkfe/8KfFkya6ZXmELbJDYljpnBvrZRQFrSLOOlthLjrgfJAQVlrEKsfRbxkhpZnDvRVoCmRTiEgsuK2xtQ82qdjyJWsONwa/UPgGw6eQ9rP+MKBTAiVzQGvH1P8rhYy5iHFA1E70FNQ6UiWfxdwFG/khCrywSeDgBdAz5IyckTTrX8tKq3q5Qi4N7eEP10UGa7CzeCz6TOOHrCXxHduIvvFIGxNaW1NWmzso52dg1N1AI+zT/vR+ScAyat5/3T0OAXaDM4EYeo6zjk01saAumHVqH2cCWAaJcf9NCdou4owkHbF+nlsJtgcsjUvavLig9JjfYvpJ7QIP2AAHkWxTmQOIRShv6M1VdRrX6X6HcdR9HIHcFUQ5bU64pT/oysudI4aDCbyOCMenSRbnU1d8dwITvCPjrbZrmca5jFBLUN/m6w28/wLCB+EDAEjy2zx4VtNNdrda5kUD88I2nWU3hrZLxZu91Y1Yf6XI5ryx9B8OzgLiHDa29s0oANIs2IhMh3ls4j2RowoAIMH3oDIgB00duT0j0Ah8QK1ueDpIwBDqfLpkAFD1PN4EQmnUPIzdhbiBxlHJ96rJvUu8FwnJzuTm43FxuDi43l5uDy40Xd4Mfp7/ddQB4PO81Ns3g9fT0+nr3a77dsplzZrQUQS/EczX8SntQDZnJQBuw56JNXpEz57ZVAg0xqfxsTeLL5GPauKevs8tk2s9s53ZMQt6Ng0GR5fS13o5ijgXfNPGHRck+BenjlufQJbLoyIKItoKIFOUI2/5rEm6LhclLB4kSunud8zYNwMV2I+Vm3eXyfc5h/EPsbIRxbgLPQ7zIGUOcnv8w/jky72/n4c1etslREcbwa6b23OF5HpDy/hTP41N9Uvjyjt6ld+Xx3JvPt03vfS2C3/B8iNlFIuwYUq6DkDKzcyBSmNXCy9CPxZs6Urg/h+gddzkCjY1R6eG+B+b51zwmJNujZHzgjU/QxlL0RhqAU+31rH8KRGa7iAlGGeqzKOkFvlgFzqfzo3m9W0oF+SXA0JlT8wxUwQM="
```

Comment:
We discussed this and I think we should be ok with this, since we'll likely remove this alert after we move to layering GA.

Just curious, does the multi arch build work?

Reply:
yes so there are packages that have "noarch" that should work. I didn't try them because I found the x86 version of aeskeyfind which ended up working. I could change it to a noarch one if needed but x86 should be fine for CI purposes.